app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/AgendaDetail/AgendaDetailScreen.kt

```diff
@@ -0,0 +1,265 @@
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.davidtomas.taskyapp.R
+import com.davidtomas.taskyapp.core.presentation.components.Header
+import com.davidtomas.taskyapp.coreUi.TaskyAppTheme
+import com.davidtomas.taskyapp.features.agenda.domain.model.AgendaType
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.AgendaTypeComposable
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.AttendeeItem
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.DescriptionComposable
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.NotificationTimePicker
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.PhotosComposable
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.PillsSelector
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.TimeDatePicker
+import com.davidtomas.taskyapp.features.agenda.presentation.agendaDetail.components.TitleComposable
+
+@Composable
+fun EventDetailScreen(
+    agendaType: AgendaType,
+    isEditable: Boolean
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(MaterialTheme.colorScheme.onPrimaryContainer)
+            .verticalScroll(rememberScrollState())
+    ) {
+        Header(
+            modifier = Modifier,
+            leadingComposable = {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_cancel),
+                    contentDescription = "",
+                    tint = Color.White
+                )
+            },
+            trailingComposable = {
+                if (isEditable)
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_edit_action),
+                        contentDescription = "",
+                        tint = Color.White
+
+                    )
+                else
+                    Text(
+                        text = "Save",
+                        color = MaterialTheme.colorScheme.onPrimary
+                    )
+            }
+        )
+        Column(
+            modifier = Modifier
+                .clip(RoundedCornerShape(topStart = 30.dp, topEnd = 30.dp))
+                .background(Color.White)
+                .padding(
+                    vertical = 30.dp,
+                    horizontal = 16.dp
+                )
+                .then(if (agendaType != AgendaType.EVENT) Modifier.weight(1f) else Modifier)
+        ) {
+            AgendaTypeComposable(agendaType = agendaType)
```

Comment:
The fact that this is a composable is quite duplicated, since you can quickly spot this by taking a look at the function signature or just the type of spelling + the current context.
It's comparable to calling an error PasswordValidationError.NO_UPPER_CASE_ERROR where the last ERROR doesn't add any extra information as it's already contained in the parent class.